ts/client/services/events/depositQueued.ts

```diff
@@ -13,7 +13,7 @@ export class DepositQueuedEventSyncer extends ContractEventSyncer {
     constructor(api: CoreAPI) {
         super(
             api.contracts.depositManager,
-            api.contracts.depositManager.filters.DepositQueued(null, null, null)
+            api.contracts.depositManager.filters.DepositQueued()
```

Comment:
Newer typechain/ethers versions do not require `null` filling. Will update this where appropriate in future work, and may do a tech-debt PR after public testnet to go through client/tests and update all of these.